app/services/proxy_status/tftp.rb

```diff
@@ -1,9 +1,7 @@
 module ProxyStatus
   class TFTP < Base
     def server
-      fetch_proxy_data do
-        api.bootServer
-      end
+      proxy.setting('TFTP', 'tftp_servername') || fetch_proxy_data { api.bootServer }
```

Comment:
Here, I don't understand. One or the other, or am I missing the point?

Reply:
Ok so we will remove the latter in 1.24. Marking as resolved then.

Reply:
should this be split to a separate PR for adding the tftp servername setting to the proxy?

Reply:
It can be, i took my direction from @ekohl to try to include some useful functionality as part of the pr (not just as capabilities and settings with nothing using them).  I'm open to whatever you'd prefer

Reply:
Initially I included this to test the functionality but now that I'm more involved in the release process I do think it's a good idea in order to track changes. We can also do some bikeshedding on its implementation, like if it's presented via an exposed setting we might not want to to the ip -> forward dns -> reverse dns -> ip dance and just trust that it's already the correct IP. That could also be done as part of 1.24.

Reply:
So I guess in that case it makes sense to split off the tftp change to a separate pr.